src/gen3authz/client/arborist/base.py

```diff
@@ -282,16 +283,32 @@ async def healthy(self, timeout=1):
         return response.code == 200
 
     @maybe_sync
-    async def auth_mapping(self, username):
+    async def auth_mapping(self, username: str = "", jwt: str = ""):
         """
         For given user, get mapping from the resources that this user can access
         to the actions on those resources for which they are authorized.
 
+        Args:
+            jwt (str): user's valid jwt access token
+            username (str): username
+
         Return:
             dict: response JSON from arborist
         """
-        data = {"username": username}
-        response = await self.post(self._auth_url.rstrip("/") + "/mapping", json=data)
+        assert not (username and jwt), "'username' and 'jwt' cannot both be provided"
+        if username:
+            data = {"username": username}
+            response = await self.post(
+                self._auth_url.rstrip("/") + "/mapping",
+                json=data,
+            )
+        elif jwt:
+            response = await self.post(
+                self._auth_url.rstrip("/") + "/mapping",
+                headers={"Authorization": f"bearer {jwt}"},
+            )
+        else:
+            response = await self.post(self._auth_url.rstrip("/") + "/mapping")
```

Comment:
What is expected to happen if neither of `username` or `jwt` is passed? Do we just make the POST call without any authN?

Reply:
it's an anonymous call and arborist will only return anonymous access